app/frontend/src/features/userPage/UserSummary.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { User } from "../../pb/api_pb";
+import {
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  makeStyles,
+} from "@material-ui/core";
+import {
+  CakeIcon,
+  LanguageIcon,
+  LocationIcon,
+  PersonIcon,
+  WorkIcon,
+} from "../../components/Icons";
+import UserSection from "./UserSection";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    //prevent a larger gap after UserSummary
+    //compared to other sections on mobile
+    marginBottom: theme.spacing(0),
+  },
+}));
+
+export default function UserSummary({ user }: { user: User.AsObject }) {
+  const classes = useStyles();
+  return (
+    <UserSection
+      title="Summary"
+      className={classes.root}
+      content={
```

Comment:
The usual way is to use children for this. And then not as a property but between the opening and closing tag.
```suggestion
      ><List ...................  </UserSection>
```